DataStructuresMemories/Models/DataSourceModel.swift

```diff
@@ -10,10 +10,11 @@ import Foundation
 
 class DataSourceModel: DataSource {
     func getDataStruct(at index: Int) -> DataStructEntity {
-        var dataStruct = DataStructEntity(with: "title", "description")
+        var dataStruct = DataStructEntity(with: "title", "description", "link")
```

Comment:
I would suggest to use `DataStructuresNames` (in singular!) instead of index
Doing this you easily avoid optionals and may even make it `DataStructuresNames` enumeration calculated property